lib/dma.h

```diff
@@ -164,15 +164,44 @@ bit_to_u8off(size_t val)
     return val % (CHAR_BIT);
 }
 
+static inline void
+u8_set_all(uint8_t *bm)
+{
+    *bm = ~0;
+}
+
+static inline void
+mask_any_lower_bits(uint8_t *byte, size_t start_idx)
```

Comment:
nit, not a fan of "start_idx" here, how about u8off to match bit_to_u8off?

Reply:
ACK